internal/kafka/confluent_kafka.go

```diff
@@ -268,7 +268,8 @@ func getMessageString(message *ckafka.Message, valueDeserializer serdes.Deserial
 		info = append(info, fmt.Sprintf("Timestamp:%d", message.Timestamp.UnixMilli()))
 	}
 	if properties.PrintOffset {
-		info = append(info, fmt.Sprintf("Partition:%d Offset:%s", message.TopicPartition.Partition, message.TopicPartition.Offset.String()))
+		info = append(info, fmt.Sprintf("Partition:%d", message.TopicPartition.Partition))
+		info = append(info, fmt.Sprintf("Offset:%s", message.TopicPartition.Offset.String()))
```

Comment:
Untested, but I think we should be able to do this:
```suggestion
		info = append(info, fmt.Sprintf("Offset:%v", message.TopicPartition.Offset))
```